docs/network-flow-visibility.md

```diff
@@ -171,6 +189,118 @@ through [Antrea Agent apiserver endpoint](prometheus-integration.md):
+  externalFlowCollectorAddr: "192.168.86.86:4739:tcp"
+  
+  # Provide flow export interval as a duration string. This determines how often the flow aggregator exports flow
+  # records to the flow collector.
+  # Flow export interval should be greater than or equal to 1s (one second).
+  # Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
+  #flowExportInterval: 60s
+  
+  # Provide the transport protocol for the flow aggregator collecting process, which is tcp or udp.
+  #aggregatorTransportProtocol: "tcp"
+```
+
+Please note that the default values for `flowExportInterval` and `aggregatorTransportProtocol`
+parameters are set to `60s` and `tcp`, respectively. Please modify them as per your
+requirements.
+
+### IPFIX Information Elements (IEs) in an Aggregated Flow Record
+
+In addition to IPFIX information elements provided in the [above section](#ipfix-information-elements-ies-in-a-flow-record),
+the Flow Aggregator adds following fields to the flow records.
+
+#### IEs from IANA-assigned IE registry
+
+| IPFIX Information Element  | Enterprise ID | Field ID | Type        |
+|----------------------------|---------------|----------|-------------|
+| originalExporterIPv4Address|      0        |   403    | ipv4Address |
+| originalObservationDomainId|      0        |   151    | unsigned32  |
+
+#### IEs from Antrea IE Registry
+
+| IPFIX Information Element                 | Enterprise ID | Field ID | Type        |
+|-------------------------------------------|---------------|----------|-------------|
+| packetTotalCountFromSourceNode            | 56506         | 120      | unsigned64  |
+| octetTotalCountFromSourceNode             | 56506         | 121      | unsigned64  |
+| packetDeltaCountFromSourceNode            | 56506         | 122      | unsigned64  |
+| octetDeltaCountFromSourceNode             | 56506         | 123      | unsigned64  |
+| reversePacketTotalCountFromSourceNode     | 56506         | 124      | unsigned64  |
+| reverseOctetTotalCountFromSourceNode      | 56506         | 125      | unsigned64  |
+| reversePacketDeltaCountFromSourceNode     | 56506         | 126      | unsigned64  |
+| reverseOctetDeltaCountFromSourceNode      | 56506         | 127      | unsigned64  |
+| packetTotalCountFromDestinationNode       | 56506         | 128      | unsigned64  |
+| octetTotalCountFromDestinationNode        | 56506         | 129      | unsigned64  |
+| packetDeltaCountFromDestinationNode       | 56506         | 130      | unsigned64  |
+| octetDeltaCountFromDestinationNode        | 56506         | 131      | unsigned64  |
+| reversePacketTotalCountFromDestinationNode| 56506         | 132      | unsigned64  |
+| reverseOctetTotalCountFromDestinationNode | 56506         | 133      | unsigned64  |
+| reversePacketDeltaCountFromDestinationNode| 56506         | 134      | unsigned64  |
+| reverseOctetDeltaCountFromDestinationNode | 56506         | 135      | unsigned64  |
```

Comment:
This is actually the first time I am looking at this, and I am actually struggling a bit to understand what these do?

How are they different from existing IEs (packetTotalCount, reversePacketTotalCount, etc)? What does `reversePacketTotalCountFromSourceNode` mean?

Reply:
For inter-Node flows, there is one stream of flow records from the source Node and another stream from the destination node. Aggregating stats across these two streams is not straight forward and they may not be similar in some scenarios, where there are intermittent losses or some sort of processing bottleneck at receiver.

To avoid loosing information,`*SourceNode` fields have aggregated statistics from the first stream of flow records and `*DestinationNode` fields have aggregated statistics from the second stream of flow records.

Reply:
Is this standard in IPFIX mediation? If not, are we sure this is the best way to handle this case?

Anyway, it is independent of this PR so we can discuss further later.